pytorch_toolkit/formula_recognition/configs/demo_config.yml

```diff
@@ -0,0 +1,23 @@
+backbone_config:
+  arch: resnext101_32x8d
+  disable_layer_3: true
+  disable_layer_4: true
+  enable_last_conv: false
+  in_lstm_ch: 512
+backbone_type: resnet
+device: cuda
+head:
+  beam_width: 0
+  dec_rnn_h: 512
+  emb_size: 80
+  enc_rnn_h: 256
+  in_lstm_ch: 512
+  max_len: 256
+  n_layer: 1
+input_images:
+- /home/vloginov/work/im2latex/medium_v2/images_processed/6124.png
```

Comment:
don't use local paths

Reply:
:+1:

Vladimir, please, use relative paths **everywhere**: you always can create a special folder for an experiment and make soft links in the folder, pointing to the proper places on your desktop

Reply:
All local paths fixed